src/components/xcpl_comps_nuopc/xlnd/src/lnd_comp_nuopc.F90

```diff
@@ -263,6 +263,7 @@ subroutine InitializeAdvertise(gcomp, importState, exportState, clock, rc)
        call fld_list_add(fldsToLnd_num, fldsToLnd, 'Sa_ptem'      )
        call fld_list_add(fldsToLnd_num, fldsToLnd, 'Sa_pbot'      )
        call fld_list_add(fldsToLnd_num, fldsToLnd, 'Sa_tbot'      )
+       call fld_list_add(fldsToLnd_num, fldsToLnd, 'Sa_uovern'    )
```

Comment:
Same comment as for xatm.

Reply:
I have update this in my recent commit. Thanks!